NSS-Spontaneous-Spoonbills/NSS-Spontaneous-Spoonbills
# Added serializers for four models

## Description of Proposed Changes

* Added Serializers and views for four tables:
  * Cust_Order
  * Employment_Dates
  * Ordered_Products
  * Payment_Options
* Deleted old SQL and added a potentially useful script (DataCheck)
* Updated the README

## Steps to Test

1. Run virtual environment
1. Start django app
1. Navigate to http://127.0.0.1:8000/bangazonapi/[tablename]
  * There are four tables to test
1. Verify that you can see the data from the database

```sh
git fetch --all
git checkout EM_SerialViews
```

## Impacted Areas in Application

**New Files**
bangazonapi/serializers/Cust_Order_Serializer.py
bangazonapi/serializers/Employment_Dates_Serializer.py
bangazonapi/serializers/Ordered_Products_Serializer.py
bangazonapi/serializers/Payment_Options_Serializer.py
views/Cust_Order_View.py
views/Employment_Dates_View.py
views/Ordered_Products_View.py
views/Payment_Options_View.py

**Modified Files**
README.md
bangazon_database/dataCheck.sql
bangazonapi/migrations/0001_initial.py
bangazonapi/serializers/__init__.py
bangazonapi/urls.py
views/__init__.py

**Deleted Files**
bangazon_database/OrdersTestData.sql
bangazon_database/reset_database.sql
bangazonapi/views.py

## Mentions @username

@obscure-shadow @JacobAndrewSmith92
